ui.tests/test-module/specs/datepicker/datepicker.runtime.spec.js

```diff
@@ -184,6 +184,11 @@ describe("Form Runtime with Date Picker", () => {
             cy.get(`#${datePicker7}`).find("input").focus().should("have.value","2/8/2023");
 
         });
+
+        // check clear option
+        cy.get(`#${datePicker7}`).find(".datepicker-calendar-icon").should("be.visible").click().then(() => {
+            cy.get(".dp-clear").click();
```

Comment:
please add another statement where post clicking clear button, the value of date picker should be removed and value should be empty.